prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -803,6 +803,29 @@ public async Task<ActionResult<Site>> DeclineSite(int siteId)
             return Ok(ApiResponse.Result(updatedSite));
         }
 
+        // PUT: api/Sites/5/enable-editing
+        /// <summary>
+        /// Enable editing a site
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpPut("{siteId}/enable-editing", Name = nameof(EnableEditingSite))]
+        [Authorize(Roles = Roles.EditSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status204NoContent)]
```

Comment:
This method returns 200 OK, but this says 204 No Content. Do you need to return the site data? Seems reasonable to change line 826 to `NoContent()`.

Reply:
The returned updatedSite can be used to reflect the submission date changed to empty in the site registration mat-table.  I think it's more clear to use the returned site from webapi rather than client side code to modify the submission date, hence business logic only resides on server side. Will change 204 to 200.